app/views/pages/home.html.erb

```diff
@@ -5,9 +5,9 @@
 <div class="row">
     <div class="col-md-offset-3 col-md-6">
       <div class="well">
-        <h4 class="text-center">Sign up for free to gain access to our community of mentors and mentees.</h4>
+        <h4 class="text-center">Sign up to gain FREE access to our community of mentors, mentees and internships.</h4>
```

Comment:
I would recommend not leaving out the oxford comma here. `mentors, mentees, and internships`